pycode/memilio-epidata/memilio/epidata/getPopulationData.py

```diff
@@ -91,6 +93,41 @@ def read_population_data(username, password, read_data, directory):
     return df_pop_raw
 
 
+def read_credentials(username, password):
+    '''! Reads credentials for regionalstatistik.de (needed for dowload).
+
+    A connfig file inside the epidata folder is written or read with following format
+    [CREDENTIALS]
+    Username = XXXXX
+    Password = XXXXX
+
+    '''
```

Comment:
```suggestion

def manage_credentials(username, password):
    '''! Manages credentials for regionalstatistik.de (needed for download).

    A config file inside the epidata folder is either written (if not existent yet) with input from user or read with following format
    [CREDENTIALS]
    Username = XXXXX
    Password = XXXXX
    
    @param username Username to sign in at regionalstatistik.de. 
    @param password Password to sign in at regionalstatistik.de.
    @return Username and password to sign in at regionalstatistik.de. 
    '''
```
I suggested to change the name of the function to manage_credentials because the function does more than reading the credentials. Furthermore I would extend the documentation.